src/components/Pairings/Pairings.js

```diff
@@ -1,8 +1,49 @@
-import React from "react";
+import React, { useContext } from "react";
+import { UserContext } from '../../UserContext';
+import { gql, useQuery } from '@apollo/client';
+
+const GET_PAIRINGS = gql`
+  {
+    getPairings(userId: "1") {
+      dateTime
+      cancelled
+    }
+  }
+`;
+
+const noPairings = (data) => (
+  data.getPairings.length === 0
+)
+
+const renderPairings = (data) => (  
+  data.getPairings.map((pairing) => {
+    console.log(noPairings(data));
+    return (
+      <div>
+        {noPairings(data) === true ? (
```

Comment:
@damwhit would you be able to help me understand why I'm not able to hit this conditional? I tried it without the `=== true` syntax as well. The console log on like 20 is returning true as expected but it's not rendering `No pairings` on the line below.

Reply:
That is interesting. One thing to note is you can do a normal if else and return some markup that way instead of returning the one piece with the nested ternary